Add Arduino sample documentation (#1159)
Azure/iotedge-lorawan-starterkit: docs/tools/arduino-samples.md

# Arduino Samples

Arduino samples can be found on the Arduino folder in the repository root. This code is built to be used for the [Seeduino LoRaWan](https://wiki.seeedstudio.com/Seeeduino_LoRAWAN/) but can serve as reference for other platforms. We have region based examples for OTAA and ABP scenarios. Please follow the [documentation](https://wiki.seeedstudio.com/Seeed_Arduino_Boards/) to set up the device

## Accessing the library files locally

> There are multuple scenarios where we would want to have the librairies be modified (listed here below). To access the local installation open your arduino IDE, go to File -> Preferences and click on the line after "more preference can be edited directly in the file".

Reviewer: ```suggestion
There are multiple scenarios where we would want to have the libraries be modified (listed here below). To access the local installation, open your Arduino IDE, go to File -> Preferences and click on the line after "more preference can be edited directly in the file".
```

![arduino files location](../images/arduinolibrarycustomsetup.png)

> Go next to the path "packages/Seeeduino/hardware/samd/1.8.2/libraries/LoRaWan and open LoRaWan.cpp and LoRaWan.h in a text editor.

Reviewer: ```suggestion
Go next to the path `packages/Seeeduino/hardware/samd/1.8.2/libraries/LoRaWan` and open `LoRaWan.cpp` and `LoRaWan.h` in a text editor.
```

Author: Thank you for the suggestions!

### CN library changes

To run our China samples some changes to the library need to be done as described below.

paste the following in the LoRaWan.h file:

``` C

>         void setChannelON(unsigned char channel);

Reviewer: ```suggestion
void setChannelON(unsigned char channel);
```

>         void setChannelOFF(unsigned char channel);

Reviewer: ```suggestion
void setChannelOFF(unsigned char channel);
```
